pkgs/test_api/lib/src/frontend/fake.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Extend or mixin this class to mark the implementation as a [Fake].
```

Comment:
Based on comments like https://github.com/dart-lang/language/issues/1068#issuecomment-656923890 I wonder if we should rephrase this to drop the encouragement of using it as a mixin. I also think it could maybe use a different phrasing in general.

```suggestion
/// A stand-in for another object which cannot be used except for specifically overridden methods.
```

Reply:
Done